src/material/slider/slider.ts

```diff
@@ -233,16 +228,16 @@ export class MatSlider extends _MatSliderBase
 
   /** Value of the slider. */
   @Input()
-  get value(): number | null {
+  get value(): number {
     // If the value needs to be read and it is still uninitialized, initialize it to the min.
     if (this._value === null) {
       this.value = this._min;
     }
-    return this._value;
+    return this._value || 0;
```

Comment:
If 0 is not in the range of the min and max, won't this cause errors? E.g. for a slider with `min: -200` and `max: -100`

Reply:
We have some tests that seems like they allow cases like that. The `|| 0` here is necessary, because TS can't infer that the `_value` is guaranteed to be a number due to the `this.value = this._min;` above. I'll change it to a type cast instead.